### PR TITLE
[rlottie]: add new port

### PR DESCRIPTION
**Pull request by SHIINASAMA**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html).
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

I accidentally deleted the original repository, which caused the [process](https://github.com/microsoft/vcpkg/pull/41567) to be terminated, and am now relaunching the pr based on the original content.

### CODE REVIEW

**Comment by jimwang118** [comment]
Has the `vcpkg.patch` patch been submitted upstream?

File: ports/rlottie/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/rlottie")
+vcpkg_fixup_pkgconfig()
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING" "${SOURCE_PATH}/AUTHORS")
```
Comment (jimwang118) [inline ports/rlottie/portfile.cmake:24]: Please add a blank line at the end of the file.

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Comment by SHIINASAMA** [comment]
@jimwang118 These patches come from the rlottie community, not directly from upstream, and they help to separate vendor dependencies, as well as adapt to a wider range of architectures and platforms.

Here's the old conversation https://github.com/microsoft/vcpkg/pull/41567

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
Usage test passed with x64-windows triplet.

**Pull request merged**